packages/embeds/embed-core/src/Inline/inline.ts

```diff
@@ -1,42 +1,43 @@
+import { EmbedElement } from "../EmbedElement";
 import loaderCss from "../loader.css?inline";
 import { getErrorString } from "../utils";
-import inlineHtml from "./inlineHtml";
+import inlineHtml, { getSkeletonData } from "./inlineHtml";
 
-export class Inline extends HTMLElement {
+export class Inline extends EmbedElement {
```

Comment:
Extending from EmbedElement that would have common functionalities for all EmbedElements like skeleton loader